src/SidePanel/TransmitSetupView.tsx

```diff
@@ -14,33 +14,37 @@ import { getIsRunning } from '../reducers/testReducer';
 import { fromPCA } from '../utils/boards';
 
 const TxPowerView = () => {
-    const txPowerIdx = useSelector(getTxPower);
+    const txPower = useSelector(getTxPower);
     const boardType = useSelector(getBoard);
     const isRunning = useSelector(getIsRunning);
     const dispatch = useDispatch();
 
     const dBmValues = fromPCA(boardType).txPower;
 
     useEffect(() => {
-        if (txPowerIdx === -1) {
-            dispatch(txPowerChanged(dBmValues.length - 1));
+        if (!dBmValues.includes(txPower)) {
+            dispatch(
+                txPowerChanged(
+                    dBmValues.includes(0) ? 0 : dBmValues[dBmValues.length / 2]
+                )
+            );
```

Comment:
If the intent see is to find some default close to the middle I thing we should make the default explicit per device!

Reply:
I think this part is kinda irrelevant and won't really be hit. It would require a device to not support 0 dBm
Alternatively I could just take the very first or last value in the dbmvalue array